### ISSUE TITLE
Bug - Lists Hover- only reading one child

### ISSUE THREAD

**Issue by twblack88** [makerdao/community-portal#136]
ugh, after approving a PR and everything. I found an interesting hover bug

![I make a list with a couple of children ](https://i.imgur.com/JVRUHJe.png)

Then on localhost something decided only one of the children was [good enough for roll over highlighting](https://imgur.com/a/AQ1s5ng). 

```
<List>

- [Writing style guide](/work_with_us/content/writing-style-guide) 
    – How to write like a Maker contributor
- [Editorial standards](/work_with_us/content/reviewer-guide) 
    – What to look out for when reviewing work
- [Visual style guide](/work_with_us/content/visual-style-guide) 
    – How to look like Comm-Dev at Maker
- <Link to="https://www.notion.so/makerdao/Maker-Brand-ac517c82ff9a43089d0db5bb2ee045a4">Design files and assets</Link> 
    – Official Maker and Dai logos, badges, and product screenshots from the Foundation design team.

</List>
```

**Comment by MaximumCrash** [comment]
@twblack88 Not a bug. Just another case of Markdown being picky. Here's what you want to write to fix your issue:

```
<List>

- [Writing style guide](/work_with_us/content/writing-style-guide) 
	- How to write like a Maker contributor


- [Editorial standards](/work_with_us/content/reviewer-guide) 
	- What to look out for when reviewing work


- [Visual style guide](/work_with_us/content/visual-style-guide) 
	- How to look like Comm-Dev at Maker


- <Link to="https://www.notion.so/makerdao/Maker-Brand-ac517c82ff9a43089d0db5bb2ee045a4">Design files and assets</Link> 
Official Maker and Dai logos, badges, and product screenshots from the Foundation design team.

</List>
```

The things to notice here is: I use 1 tab, then a dash "-", then a space. <- This works for non <Link> elements

In the case of your notion link the reason it doesn't behave the same, is the same reason we have frament crash issues when not putting a newline at the end of a list (or any other component). 

It will render your last link like this:
```
<ul> <- List container

<li>
<Link>...</Link>
<div> <- Sub content
- Official Maker and Dai logos, badges.  <- Notice the tab is written in there.
</div>

</ul>
```

The reason for this is to support some kind of consistency in cases where MD doesn't recognize it as an (li) list element. 

The fixes in this case are:

1. Use the Markdown spec link. <Link> is the equivalent of `[Link Text](Link Url)` in markdown. (Just like how CTAs are blockquotes):
```
<List>

- [Design files and assets](https://www.notion.so/makerdao/Maker-Brand-ac517c82ff9a43089d0db5bb2ee045a4) 
	- Official Maker and Dai logos, badges, and product screenshots from the Foundation design team.

</List>
```

2. If you must use <Link> in a certain case, don't use the "-" for the child content:

```
<List>

- <Link to="https://www.notion.so/makerdao/Maker-Brand-ac517c82ff9a43089d0db5bb2ee045a4">Design files and assets</Link> 
Official Maker and Dai logos, badges, and product screenshots from the Foundation design team.

</List>
```

**Comment by MaximumCrash** [comment]
I will say though that writing them naturally the way you do is something I would like to support. So I will use this as a nice to have since it's lower priority right now.

**Comment by MaximumCrash** [comment]
@twblack88 The list scripting spec has changed. Please see the comments on #199 about this. 

Closed by #199
